### DISCUSSION TITLE
[V2] ability to add lazy load to image/video field

### DISCUSSION THREAD

**Discussion by lotarbo** [awcodes/filament-curator#272] [Ideas]
Hello, i think a have a problem with performance in filament admin panel.
I have Builder field with many blocks, inside blocks i have curator field with image/video file. End regular page in admin panel uses 8-14 mb network. So it seems loading media can slow down page load.
I think about ability to add lazy prop to image and `preload="none"` to video maybe

**Comment by awcodes** [comment]
Possibly, I look into it, but I'm really not trying to add anything new to v2. :)

**Reply by lotarbo** [reply]
@awcodes i understand, and it can be feature in v3) but i still use v2 because filament v3 and all deps not stable yet(
have plan to upgrade filament but with stable deps)

**Reply by awcodes** [reply]
Totally understand. Due to how loading lazy works though and that it shouldn't be used on elements above the page fold. You'd still have to manually add it to each Picker that you think will be below the page fold. But I'll see what I can do.

**Comment by awcodes** [comment]
This is available in latest release. cheers.

**Reply by lotarbo** [reply]
@awcodes thank you
